.github/workflows/release.yml

```diff
@@ -34,6 +34,8 @@ jobs:
         with:
           arguments: assemble
         env:
+          GITHUB_ACTOR: ${{ secrets.GITHUB_ACTOR }}
```

Comment:
Is `GITHUB_ACTOR` a secret?

Reply:
This is a default available value that represents the principal running the github action.  I think the theory is that because GH_TOKEN will be a PAT, the username is not really needed and only there to satisfy maven.

Reply:
Well, I don't think `GITHUB_ACTOR` is  secret. It is an environment variable always set for github actions. https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables

Reply:
the username doesn't mater in my testing, so i'm going to remove it.